auto_backup/models/db_backup.py

```diff
@@ -4,25 +4,27 @@
 # © 2016 Grupo ESOC Ingeniería de Servicios, S.L.U. - Jairo Llopis
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/gpl.html).
 
+from contextlib import contextmanager
+from datetime import datetime, timedelta
+from glob import iglob
+import logging
 import os
 import shutil
 import tempfile
 import traceback
-from contextlib import contextmanager
-from datetime import datetime, timedelta
-from glob import iglob
-from openerp import exceptions, models, fields, api, _, tools
+
+from openerp import api, exceptions, fields, models, tools, _
 from openerp.service import db
-import logging
+
 _logger = logging.getLogger(__name__)
 try:
     import pysftp
 except ImportError:
-    _logger.warning('Cannot import pysftp')
-
+    _logger.warning("Cannot import pysftp")
```

Comment:
Your rewrite is not efficient because you are changing things that don't need to be changed. If you find an error, please make a PR changing only the lines of the error.